src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
+    private Machine truck = new Truck();
+    private Machine bulldozer = new Bulldozer();
+    private Machine excavator = new Excavator();
+    private Machine[] machines = {truck, bulldozer, excavator};
```

Comment:
Do not declare `Machine` variables at the class-level, because you can explicitly create them in the array initialization. also this array should be local variable.